daemon/logger/jsonfilelog/jsonlog/jsonlogbytes_test.go

```diff
@@ -29,6 +29,8 @@ func TestJSONLogsMarshalJSONBuf(t *testing.T) {
 		{Log: []byte{0x7F}}:            `^{\"log\":\"\x7f\",\"time\":`,
 		// with raw attributes
 		{Log: []byte("A log line"), RawAttrs: []byte(`{"hello":"world","value":1234}`)}: `^{\"log\":\"A log line\",\"attrs\":{\"hello\":\"world\",\"value\":1234},\"time\":`,
+		// with Tag set
+		{Log: []byte("A log line with tag"), RawAttrs: []byte(`{"hello":"world","value":1234}`)}: `^{\"log\":\"A log line with tag\",\"attrs\":{\"hello\":\"world\",\"value\":1234},\"time\":`,
```

Comment:
@yongtang I don't understand how the json string in RawAttrs gets merged into the expected output json: what if I put garbage in RawAttrs? Where is RawAttrs set?

Reply:
@tiborvass Thanks for the review. The RawAttrs was set through:
https://github.com/yongtang/docker/blob/dd5b0f3a4867c253aa65a19fed12a795571ce7db/daemon/logger/jsonfilelog/jsonfilelog.go#L79-L90

The `attrs` was marshaled into json string then passed to `RawAttrs`:
https://github.com/yongtang/docker/blob/dd5b0f3a4867c253aa65a19fed12a795571ce7db/daemon/logger/jsonfilelog/jsonfilelog.go#L118-L128

The RawAttrs previously already handles user defined `attrs`. This PR attach the `tag` to the `attrs` so it is not very visible in the diffs of this PR.